readme.md

```diff
@@ -47,10 +47,10 @@ Gradle:
       jcenter()
     } 
     dependencies {
-      compile "com.palantir.remoting:http-clients:0.1.2"
-      compile "com.palantir.remoting:http-servers:0.1.2"
-      compile "com.palantir.remoting:trust-stores:0.1.2"
-      compile "com.palantir.remoting:error-handling:0.1.2"
+      compile "com.palantir.remoting:http-clients:$version"
+      compile "com.palantir.remoting:http-servers:$version"
+      compile "com.palantir.remoting:ssl-config:$version"
+      compile "com.palantir.remoting:error-handling:$version"
     }
```

Comment:
since 2-way auth is the main new feature here, maybe it would be worth putting an illustrative example here for how to set up dropwizard/feign with 2-way ssl?